src/sources/forus-platform/js/angular-1/components/Modals/ModalVouchersUploadComponent.js

```diff
@@ -182,6 +182,8 @@ const ModalVouchersUploadComponent = function(
                         });
 
                         row.note = row.note || this.defaultNote(row);
```

Comment:
@dev-rminds @aghimpu not going to show warning in this pr, right?

Reply:
discussed with devs - will be in separate issue, where we will turn old naming off